ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/management-ssh.yml

```diff
@@ -15,3 +15,6 @@ management_ssh:
   client_alive:
     count_max: 42
     interval: 666
+  hostkey:
```

Comment:
please also add the hostkey.server... that was already in the schema. Seems like we have missed testing that. Thanks!

Reply:
Sure, I will add it.

Reply:
Modified the schema and template with test case

Reply:
The test for `hostkey.server` is already there in `management-ssh-custom-cipher.yml`.
I have added a `server_cert_key` to the data-model and test-case for it.